### ISSUE TITLE
Javascript minify line feed problem solve

### ISSUE THREAD

**Issue by sedattanir** [matthiasmullie/minify#243]
Javascript minify is working but there is a problem. Minify isn't remove line feed every line. If you want to remove, Do this:
1- Open "vendor\matthiasmullie\minify\src" file.
2- Find this line "$content = preg_replace('/\n+/', "\n", $content);"
3- Change "$content = preg_replace('/\n+/', "", $content);"

And then this is working :)

**Comment by matthiasmullie** [comment]
Hi @sedattanir 

It is correct that not every line feed is being removed. In some instances, a line feed is actually required if a statement has not been terminated with `;`. So minify will play it safe and only replace the line feeds that are 100% safe to remove. In other cases, it can't just be stripped: it would sometimes have to be replaced by a `;` (but since both are just 1 character, there's no point replacing those anyway)

There may be a few leftover newlines that serve no purpose, where Minify played it too safe (where it is hard to detect if it could safely be removed), but those are few.

In short: I definitely wouldn't recommend all newlines be stripped, unless you're very careful never to forget a `;` somewhere...
